File: models/OtherRevenue.js
import mongoose  from "mongoose"

const OtherRevenueSchema = new mongoose.Schema({
    userId: { type: mongoose.Schema.Types.ObjectId, ref: 'User'},
    description: { type:  String, required: true}, 
    amount: { type: Number, required: true}, 
    created: { type: String, required: true }, 
   
})

const OtherRevenue = mongoose.model('OtherRevenue', OtherRevenueSchema)

export default OtherRevenue

File: app/src/main/java/com/example/test/models/OtherRevenue.kt
package com.example.test.models

import java.io.Serializable

data class OtherRevenue(
    var userId:String,
    var description: String,
    var amount: Double,
    var created:String
): Serializable {
    val _id: String?= null
    constructor(
        _id:String,
        userId:String,
        description: String,
        amount: Double,
        created: String,
    ) : this(
        userId,
        description,
        amount,
        created)
}

